WoWsShipBuilder.Desktop/Features/BlazorWebView/DesktopLayout.razor

```diff
@@ -13,13 +19,41 @@
         </CascadingValue>
     </MudMainContent>
     <footer style="margin-top: 16px;">
-        <ShipBuilderFooter/>
+        <ShipBuilderFooter IsDarkTheme="isDarkTheme"/>
```

Comment:
this is ok, a nicer solution would have been to also solve this entirely through css. But since this component is only ever used from the main layout, it's fine